backend/src/main/java/ca/gov/dtsstn/cdcp/api/data/entity/ConfirmationCodeEntity.java

```diff
@@ -0,0 +1,105 @@
+package ca.gov.dtsstn.cdcp.api.data.entity;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import java.time.Instant;
+import org.immutables.builder.Builder;
+import org.springframework.core.style.ToStringCreator;
+import org.springframework.lang.Nullable;
+@Entity(name = "ConfirmationCode")
+@SuppressWarnings({ "serial" })
+public class ConfirmationCodeEntity extends AbstractEntity {
+    
+    @Column(length = 9, nullable = false, updatable = false)
+	private String userId;
+
+	@Column(length = 50, nullable = false)
+	private String email;
+
+    @Column(length = 50, nullable = false)
+	private String confirmationCode;
+
+    @Column(length = 50, nullable = false)
+	private Instant codeCreatedDate;
+
+    @Column(length = 50, nullable = false)
+	private Instant codeExpiryDate;
```

Comment:
I think this should be called `expiryDate`

Reply:
field renamed